### PR TITLE
Create initial version of LSP-dockerfile

### PR DESCRIPTION
**Pull request by rchl**
@jwortmann

### CODE REVIEW

File: plugin.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+def plugin_unloaded():
+    server.cleanup()
+
+
+def is_node_installed():
+    return shutil.which('node') is not None
+
+
+class LspVuePlugin(LanguageHandler):
```
Comment (rwols) [inline plugin.py:29]: The name of this language server is not "Vue"

File: plugin.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        configuration = self.read_configuration()
+
+        default_configuration = {
+            'enabled': True,
+            'command': ['node', server.binary_path, '--stdio'],
+        }
+
+        default_configuration.update(configuration)
+
+        return read_client_config(self.name, default_configuration)
```
Comment (rwols) [inline plugin.py:50]: These lines are starting to look very familiar :). Consider creating a `NodeBasedLanguageHandler` base class or something in `lsp_utils`.

Comment (rchl) [reply plugin.py:50]: Good idea. Hopefully not too challenging to have it flexible enough.

Comment (rchl) [reply plugin.py:50]: We'll handle that afterward.

**Pull request merged**